src/utils/strings.spec.ts

```diff
@@ -0,0 +1,15 @@
+import {hasProtocol} from './strings';
+
+describe('strings', () => {
+  describe('hasProtocol', () => {
+    it('should return true if the url has a protocol', () => {
+      expect(hasProtocol('https://www.google.com')).toBe(true);
+      expect(hasProtocol('http://www.google.com')).toBe(true);
```

Comment:
We could check other protocols too.
```suggestion
      expect(hasProtocol('http://www.google.com')).toBe(true);
      expect(hasProtocol('ftp://www.google.com')).toBe(true);
```